cluster/log-dump.sh

```diff
@@ -152,10 +159,19 @@ function kubectl-guess-nodes() {
   kubectl get node -ojsonpath --template="${ips_and_images}" | grep -v kube-apiserver | cut -f1 -d" "
 }
 
-function dump_master() {
-  local master_name
+function awscli-instances() {
+  local -r role=$1
+  for region in "${awscli_regions[@]}"; do
+    aws ec2 describe-instances --region "${region}" --filter "Name=tag-value,Values=$(kubectl config current-context)" "Name=instance-state-name,Values=running" "Name=tag-key,Values=k8s.io/role/${role}" | jq -r '.["Reservations"][]["Instances"][]["PublicDnsName"]'
```

Comment:
Thanks! See below, but yeah, I just tested this with `aws ec2 describe-instances --filter "Name=tag:KubernetesCluster,Values=$(kubectl config current-context)" "Name=tag:k8s.io/role/node,Values=1" --query "Reservations[].Instances[].PublicDnsName" --output text` and it works as expected. I'll incorporate this into the new custom function.